pkg/e2e/operators/cloudingress/apischeme_cr.go

```diff
@@ -13,24 +13,58 @@ import (
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/rest"
 )
 
 var _ = ginkgo.Describe(constants.SuiteOperators+TestPrefix, func() {
 	h := helper.New()
-	testCRapiSchemesPresent(h)
-	testDaCRapischemes(h)
-	testCRapischemes(h)
+	ginkgo.Context("apischeme", func() {
+		ginkgo.It("apischemes CR instance must be present on cluster", func() {
```

Comment:
This should be in a wait loop.. because this is pushed out from hive, there can be a slight delay before it's pushed out.